ganga/GangaCore/GPIDev/Lib/Job/Job.py

```diff
@@ -729,13 +812,18 @@ def getSubJobStatuses(self):
 
 	return stats
 
+    def returnSubjobStatuses(self):
+        if hasattr(self.info, 'subjob_statuses'):
```

Comment:
I don't think that the `info` part is in the index file. So will this not trigger loading of the full job?

Reply:
I think I agree with Ulrik here, and upon inspection I realize I forgot to add a test to make sure the job summary is safe for lazy loading so this can be broken and pass the integration suite.

Reply:
@mesmith75 does this PR pass if you try to run #1346?

Reply:
@rob-c Looks to be.

Reply:
Ah, OK, I think I see the code that checks for the lazy loading being broken. this looks like a quite cool extension :+1: